### PR TITLE
[IT-835] initial commit

### PR DESCRIPTION
**Pull request by zaro0508**
A packer template for packer ami projects

### CODE REVIEW

File: src/template.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    {
+      "ansible_env_vars": [
+        "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'"
+      ],
+      "playbook_file": "playbook.yaml",
+      "type": "ansible"
+    }
+  ],
+  "variables": {
+    "AmiUsers": "563295687221,055273631518,804034162148,237179673806",
```
Comment (zaro0508) [inline src/template.json]: wondering if we should set this to `all` so that it's publicly accessible?

File: src/template.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "Department": "Platform",
+    "ImageName": "{{env `ImageName`}}",
+    "InstanceType": "t3.nano",
+    "OwnerEmail": "joe.smith@sagebase.org",
+    "Project": "Infrastructure",
+    "SnapshotUsers": "563295687221,055273631518,804034162148,237179673806",
```
Comment (zaro0508) [inline src/template.json]: same here, maybe make it publicly accessible?

File: src/template.json (hunk 1 above)
Comment (thomasyu888) [reply src/template.json]: Did we want amis to be public?

Comment (zaro0508) [reply src/template.json]: hey, isn't that what i asked?

Comment (zaro0508) [reply src/template.json]: we decided yes with explicit description in README

File: README.md [hunk 3]
```diff
@@ -1,2 +1,114 @@
...
+source_profile = jsmith@imagecentral
+```
+
+Now you will be able to build an image and deploy it to Imagecentral.
+
+### Manual AMI Build
+If you would like to test building an AMI run:
+```
+cd src
+packer build -var AwsProfile=packer-service-imagecentral -var AwsRegion=us-east-1 -var ImageName=my-test-image src/template.json
```
Comment (zaro0508) [inline README.md]: add `-var PACKER_LOG=1`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,114 @@
...
+packer validate -var 'ImageName=my-test-image' template.json
+```
+
+### AWS Access
+To run a build you must have an AWS account and access to EC2.
+
+* Request an IAM account in [Imagecentral](https://github.com/Sage-Bionetworks/imagecentral-infra)
+* Change password and set up MFA
+* Create a Keypair
+* Add your access code and secrety key to `~/.aws/credentials`, using a profile such as "imagecentral.jsmith"
```
Comment (zaro0508) [inline README.md]: secrety/secret

**Pull request merged**